en/micro-integrator/docs/setup/deployment/deploying_wso2_ei.md

```diff
@@ -106,7 +106,10 @@ When the nodes in the cluster need to communicate with each other, the Micro Int
     pool_options.maxWait = 60000
     pool_options.testOnBorrow = true
     ```
-
+    
+!!! Note
+    RDBMS-based coordination needs the system time of each node to be synchronized. Having them not synchronized will lead to unexpected behaviors. Please refer [EI_1758](https://github.com/wso2/docs-ei/issues/1758) for one of possible issues.
```

Comment:
```suggestion
    RDBMS-based coordination needs the system time of each node to be synchronized. Having them not synchronized will lead to unexpected behavior. Please refer [EI_1758](https://github.com/wso2/docs-ei/issues/1758) for one of the possible issues.
```

Why are we pointing to an issue in the doc repo? Shouldn't it be product-ei or somewhere similar?

Reply:
we don't have any other issue . this is here since this is moved here since this is doc related one